cypress/e2e/components/filings/addStaffFiling.cy.ts

```diff
@@ -66,7 +66,7 @@ context('Add Staff Filing', () => {
         response).as('businessComments')
     })
     cy.visitBusinessDashFor('businessInfo/bc/historical.json', undefined, false, false, undefined, allFilings, true)
-    // cy.wait(5000)
+    cy.wait(1000)
```

Comment:
I know for sure this works when waiting 5000. Trying 1000 now as it's less intrusive. I'll revert if this if it doesn't work.

Reply:
It works. This failed intermittently previously.